languages/ruby/exercises/concept/instance-variables/.meta/design.md

```diff
@@ -0,0 +1,38 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics of the Concept instance variables and "nil" in Ruby.
+
+## Learning objectives
+
+- Know that objects can store state using instance variables
+- Know that instance variables should be treated as private and to use getter and setter methods
+- Know that `nil` is a Ruby object used to represent "nothingness"
+- Know that instance variables are `nil` before they are assigned otherwise
+
+## Out of scope
+
+- Boolean values (True/False)
```

Comment:
This would be reinforcement of the idea to not state "methods that return true or false should be named with a question mark"  Mention Boolean, but not the details of Boolean yet (around line 35 of "introduction.md".